### DISCUSSION TITLE
No openchrom tag on stackoverflow

### DISCUSSION THREAD

**Discussion by brandav** [OpenChrom/openchrom#477] [Ideas]
I asked a [question on stackoverflow ](https://stackoverflow.com/questions/58250187/openchrom-ide-user-product-not-showing-up-in-eclipse-installer-simple-mode) but don't have the reputation yet to create a new tag for openchrom. Would someone who does have enough reputation do this?

**Comment by kerner1000** [comment]
Hi, many thanks for contributing. Please go ahead and post your question on stackoverflow and then link it here. We will tag it later, as soon as someone has enough reputation to do so.

**Comment by kerner1000** [comment]
Tag exists now.
https://stackoverflow.com/questions/tagged/openchrom
